tests/unit/FormInput.spec.js

```diff
@@ -71,39 +76,43 @@ describe('FormInput', () => {
     expect(wrapper.find('input').element.name).toBe(nameText);
     expect(wrapper.find('input').element.placeholder).toBe(placeholderText);
     expect(wrapper.find('input').classes('is-invalid')).toBe(true);
-    expect(wrapper.find('.invalid-feedback').exists()).toBe(true);
-    expect(wrapper.find('.invalid-feedback').isVisible()).toBe(true);
-    expect(wrapper.html()).toContain(errorText);
-    expect(wrapper.html()).toContain(requiredText);
+    expect(wrapper.find(DisplayErrors).exists()).toBe(true);
   });
 
   it('displays validation error messages when the field is invalid', () => {
-    const requiredText = 'The FormInput field is required.';
     const errorText = 'This field has an error';
     const wrapper = factory({
       name: 'FormInput',
       error: errorText,
       validation: 'required'
     });
-    
+
+    wrapper.setProps({value: "", validation: 'required'});
```

Comment:
It looks like you already set `validation: 'required'` in the factory method above. Is there a reason you need to set it again here?

Reply:
yes, I have to remove it from the factory.